osquery/tables/system/windows/system_info.cpp

```diff
@@ -55,7 +55,19 @@ QueryData genSystemInfo(QueryContext& context) {
     r["hardware_vendor"] = "-1";
     r["hardware_model"] = "-1";
   }
-  
+
+  QueryData regResults;
+  queryKey(
+      "HKEY_LOCAL_MACHINE\\"
+      "HARDWARE\\DESCRIPTION\\System\\CentralProcessor\\0\\Update Revision",
```

Comment:
@muffins, not sure why this doesn't produce results. There are no rows returned from what I can tell.

If I do a:
```sql
select * from registry where path = 'HKEY_LOCAL_MACHINE\\HARDWARE\\DESCRIPTION\\System\\CentralProcessor\\0\\Update Revision';
```

It works.

Reply:
Ah!  You are querying the value, you want to remove the `Update Revision` component, as the key you want is the `0` value, so it should be:
```
  QueryData regResults;
  queryKey(
      "HKEY_LOCAL_MACHINE\\"
      "HARDWARE\\DESCRIPTION\\System\\CentralProcessor\\0\\",
      regResults);
  for (const auto& key : regResults) {
    if (key.at("name") == "Update Revision") {
      r["cpu_microcode"] = key.at("data");
      break;
    }
  }
```

Reply:
Also looks like you'll want an include for `registry.h`:
```
#include "osquery/tables/system/windows/registry.h"
```